contracts/hackatom/src/contract.rs

```diff
@@ -168,11 +170,11 @@ fn do_release<S: Storage, A: Api, Q: Querier>(
         ctx.set_data(&[0xF0, 0x0B, 0xAA]);
         Ok(ctx.into())
     } else {
-        Err(StdError::unauthorized())
+        Err(Unauthorized {}.build())
```

Comment:
This is snafu style, right?

Reply:
Right, hackatom uses Snafu to proof the flexibility of this overall stategy. Using safu in hackatom is a bit of a bad idea since in https://github.com/CosmWasm/cosmwasm/pull/537 I wanted to see if we can reduce size or gas by migrating away from Snafu. So the Snafo demo will likely need to move to another contract.